modules/lvms-scaling-storage-of-clusters-using-cli.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-scaling-storage-of-clusters-using-cli_{context}"]
+= Scaling up the storage of clusters by using the CLI
```

Comment:
You could probably drop "by" here:
```suggestion
= Scaling up the storage of clusters using the CLI
```

Reply:
I think it would be best to use "by" to keep it consistent with other titles in the doc.